reproducibility_project/init.py

```diff
@@ -55,11 +55,11 @@ def dict_product(dd):
 g_per_cm3 = u.g / (u.cm * u.cm * u.cm)
 masses = {
     "methaneUA": [16.043] * u.amu,
-    "pentaneUA-flexible_bonds": [72.15] * u.amu,
```

Comment:
@CalCraven @daico007 
I have updated the MW of the 5 molecules based on what exactly is going into LAMMPS/GROMACS simulations. 
If this sounds okay to you guys, then I can merge this PR.